opennms-doc/guide-admin/src/asciidoc/text/rmi.adoc

```diff
@@ -22,7 +22,7 @@ ADDITIONAL_MANAGER_OPTIONS="$ADDITIONAL_MANAGER_OPTIONS -Dopennms.poller.server.
 ADDITIONAL_MANAGER_OPTIONS="$ADDITIONAL_MANAGER_OPTIONS -Djava.rmi.server.hostname=<your-server-ip-address>"
 ----
 
-This tells {opennms-product-name} to listen for RMI on port `18980`, and to listen on all interfaces.  (Originally, RMI was only used for the Remote Poller, so despite the porperty name mentioning the "opennms poller server" it applies to RMI as a whole.)  Note that you _must_ include the `-Djava.rmi.server.hostname=` option or {opennms-product-name} will accept connections on the RMI port, but not be able to complete a valid connection.
+This tells {opennms-product-name} to listen for RMI on port `18980`, and to listen on all interfaces. (Originally, RMI was only used for the deprecated Remote Poller, so despite the property name mentioning the "opennms poller server" it applies to RMI as a whole.)  Note that you _must_ include the `-Djava.rmi.server.hostname=` option or {opennms-product-name} will accept connections on the RMI port, but not be able to complete a valid connection.
```

Comment:
I think the term `deprecated` is misleading when talking about the remote poller. We still have one (better than ever)

Reply:
Changed it to 'legacy'.